packages/server/graphql/mutations/removeTeamMember.ts

```diff
@@ -31,6 +32,7 @@ export default {
     }
 
     // RESOLUTION
+    removeSlackAuths(teamMemberId, false)
```

Comment:
i feel like this call belongs in `removeTeamMember`. that way you won't have to modify removeOrgUser,